indico/modules/events/timetable/templates/pdf/_program.html

```diff
@@ -0,0 +1,265 @@
+                                    <div class="contrib-heading">
+                                        {% if not session.is_poster %}
+                                            <span class="timebox inherited-bg-color inherited-txt-color">
+                                                {{ format_interval(child.start_dt.astimezone(child.event.tzinfo), child.end_dt.astimezone(child.event.tzinfo), format='hh:mm') }}
+                                                {% if show_length_contribs %}
+                                                    ({{ (child.end_dt - child.start_dt)|format_human_timedelta(narrow=True) }})
+                                                {% endif %}
+                                            </span>
+                                        {% endif %}
+                                        <h4>{{ child.object.title }}</h4>
+                                    </div>
+                                    {% if child.type.name == 'CONTRIBUTION' %}
+                                        <div class="contrib-timeline-content" >
+                                            {{ render_nested_contrib(child, show_abstract, dont_show_poster_abstract, show_title, show_affiliation, show_length_contribs) }}
+                                        </div>
+                                    {% endif %}
+                                </li>
+                            {% endfor %}
+                        </ul>
+                    </div>
+                {% endif %}
+            </div>
+        {% endif %}
+    </div>
+{% endmacro %}
+
+{% macro render_nested_contrib(entry, show_abstract, dont_show_poster_abstract, show_title, show_affiliation, show_length_contribs=false) %}
+    {% set contrib = entry.object if entry.object else entry %}
+    {% set speakers = contrib.speakers %}
+
+    {% if speakers %}
+        <p>
+            <span class="bold">{{ ngettext('Speaker', 'Speakers', speakers|length) }}</span><br />
+            {{ render_names(speakers, show_title=show_title, show_affiliation=show_affiliation) }}
+        </p>
+    {% endif %}
+
+    {% if not contrib.inherit_location and contrib.has_location_info %}
+        <p>
+            <span class="bold">{% trans %}Location{% endtrans %}</span><br />
+            {{ render_place(contrib) }}
+        </p>
+    {% endif %}
+    {% if show_abstract and (not dont_show_poster_abstract or entry.is_poster) and contrib.description %}
+        <div class="description">
+            <span class="bold">{% trans %}Description{% endtrans %}</span>
+            <div>{{ contrib.description }}</div>
+        </div>
+    {% endif %}
+    {% if contrib.subcontributions %}
+        <div>
+            <span class="bold">{% trans %}Subcontributions{% endtrans %}</span>
+            <ul class="sub-contrib-timeline inherited-bg-color-trans inherited-border-color">
+                {% for subchild in contrib.subcontributions %}
```

Comment:
I don't want to be too nitpicky, but we typically use `subcontrib` for subcontributions. I think `subchild` is a bit less clear